R/xml_modify.R

```diff
@@ -139,6 +142,49 @@ xml_add_child.xml_nodeset <- function(.x, .value, ..., .copy = TRUE) {
 
 #' @rdname xml_replace
 #' @export
+xml_prepend_child <- function(.x, .value, ..., .copy = TRUE) {
+  UseMethod("xml_prepend_child")
+}
+
+#' @export
+xml_prepend_child.xml_node <- function(.x, .value, ..., .copy = inherits(.value, "xml_node")) {
+  children <- xml_children(.x)
```

Comment:
Should this be children or contents? What if you want to insert after a text node?